backend/layers/processing/dataset_metadata_update.py

```diff
@@ -353,28 +296,8 @@ def update_metadata(
             self.logger.error(f"Cannot find labeled H5AD artifact uri for {current_dataset_version_id}.")
             self.update_processing_status(new_dataset_version_id, DatasetStatusKey.H5AD, DatasetConversionStatus.FAILED)
 
-        if DatasetArtifactType.RDS in artifact_uris:
-            self.logger.info("Main: Starting thread for rds update")
-            rds_job = Process(
-                target=DatasetMetadataUpdater.update_rds,
-                args=(
-                    self.artifact_bucket,
-                    self.datasets_bucket,
-                    artifact_uris[DatasetArtifactType.RDS],
-                    new_artifact_key_prefix,
-                    new_dataset_version_id,
-                    metadata_update,
-                ),
-            )
-            artifact_jobs.append(rds_job)
-            rds_job.start()
-        elif current_dataset_version.status.rds_status == DatasetConversionStatus.SKIPPED:
-            self.update_processing_status(new_dataset_version_id, DatasetStatusKey.RDS, DatasetConversionStatus.SKIPPED)
-        else:
-            self.logger.error(
-                f"Cannot find RDS artifact uri for {current_dataset_version_id}, and Conversion Status is not SKIPPED."
-            )
-            self.update_processing_status(new_dataset_version_id, DatasetStatusKey.RDS, DatasetConversionStatus.FAILED)
+        # Mark all RDS conversions as skipped
```

Comment:
this seemed like the simplest backwards compatible solution

Reply:
this approach is consistent since we treat metadata updates as new versions, but we should double-check with Lattice/Brian that they're aware + cool with it

Reply:
https://czi-sci.slack.com/archives/C07AV4NU9D2/p1731535747847619